src/components/ViewSource/ViewSource.test.js

```diff
@@ -105,4 +113,44 @@ describe('ViewSource', () => {
       });
     });
   });
+
+  describe('when Instance is shared', () => {
+    beforeEach(async () => {
+      useInstance.mockReturnValue({
+        isLoading: false,
+        instance: {
+          title: 'Instance title',
+          source: `${CONSORTIUM_PREFIX}MARC`,
+        },
+      });
+
+      await act(async () => {
+        await renderWithIntl(getViewSource(), translations);
+      });
+    });
+
+    it('should display "view shared marc bibliographic record" message', () => {
+      expect(screen.getByText('Shared MARC bibliographic record')).toBeInTheDocument();
```

Comment:
You are testing by `"view shared marc bibliographic record"` but expecting `Shared MARC bibliographic record`, shouldn't you expect `view shared marc bibliographic record` here?